packages/apps/intervention/src/components/molecules/CustomerParkHeader/CustomerParkHeader.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2024 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import {View} from 'react-native';
+import {ClientProspectSearchBar, PlaceEquipmentSearchBar} from '../../atoms';
+
+const CustomerParkHeader = ({
+  setCustomer,
+  customer,
+  inService,
+  parentPlace,
+  setParentPlace,
+}) => {
+  return (
+    <View>
+      <ClientProspectSearchBar
+        required={true}
+        defaultValue={customer}
+        onChange={e => {
+          setCustomer(e);
+        }}
+      />
+      <PlaceEquipmentSearchBar
+        defaultValue={parentPlace}
+        inService={inService}
+        customerId={customer?.id}
+        onChange={e => {
+          setParentPlace(e);
+        }}
+      />
+    </View>
+  );
+};
```

Comment:
I'm not sure this is useful to create a component for that, there is no logic and no style in it. It can be moved directly to the screen